src/o365/spo/commands/apppage/apppage-update.ts

```diff
@@ -0,0 +1,110 @@
+import request from '../../../../request';
+import commands from '../../commands';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+import GlobalOptions from '../../../../GlobalOptions';
+import Utils from '../../../../Utils';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  pageName: string;
+  webPartData: string;
+}
+
+class SpoAppPageUpdateCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.APPPAGE_UPDATE}`;
+  }
+
+  public get description(): string {
+    return 'Updates the single-part app page';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const requestOptions: any = {
+      url: `${args.options.webUrl}/_api/sitepages/Pages/UpdateFullPageApp`,
+      headers: {
+        'content-type': 'application/json;odata=nometadata',
+         accept: 'application/json;odata=nometadata'
+      },
+      json: true,
+      body: {
+        serverRelativeUrl: `${Utils.getServerRelativePath(args.options.webUrl)}/SitePages/${args.options.pageName}`,
+        webPartDataAsJson: args.options.webPartData
+      }
+    };
+
+    request.post(requestOptions).then((res: any): void => {
+      cmd.log(res);
```

Comment:
That `cmd.log(res);` for the response is `odata.null: true` when success. Should we keep it here?

Reply:
I did not like it as well, but seeing how other commands have kept this out put I left it there incase this is needed. I can remove it.

Reply:
Ok lets ask @waldekmastykarz , that would be helpful. Here is my thinking .. it simple .. if I follow any kind of SOLID practices, I return information for `get, list` methods and should return void for `set, add`. However there are always exceptions, for example when the REST api  returns the newly created object on `add` it is sometimes good to have it returned, but what if it returns null: true? 

Waldelk what is your philosophical opinion on this?

Reply:
Imho, if there is nothing to output (eg. `odata.null: true`), then we shouldn't print it as it doesn't add any value. Hope it helps 🙂